### PR TITLE
fixes #16196 - fix typo in image scoped search

### PR DESCRIPTION
**Pull request by martin-ducar-gd**

### CODE REVIEW

File: test/unit/image_test.rb [hunk 1]
```diff
@@ -10,4 +10,9 @@ class ImageTest < ActiveSupport::TestCase
     image.valid? #trigger validations
     assert image.errors.messages.keys.include?(:uuid)
   end
+
+  test "image scoped search for compute_resource works" do
+    resource = compute_resources(:one)
+    assert Image.search_for("compute_resource = #{resource.name}")
```
Comment (domcleal) [inline test/unit/image_test.rb]: Better might be to check the result includes one of the images assigned to this compute resource (test/fixtures/images.yml): `assert_includes Image.search_for("compute_resource = #{resource.name}"), images(:one)`

**Comment by domcleal** [comment]
Thanks, works well - just one comment.

[test]

**Comment by timogoebel** [comment]
[test]

**Comment by tbrisker** [comment]
Merged as 96eed04df9ba23e743da161a8afe43cab8e84fcc, thanks @martin-ducar-gd!

**Pull request closed without merging**